sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.csproj

```diff
@@ -17,10 +17,12 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Azure.Data.SchemaRegistry\src\Azure.Data.SchemaRegistry.csproj" />
```

Comment:
We'll need to be very careful here.  The package is GA and we cannot release any GA updates to it with an Experimental ref.    I don't see us releasing, so we're probably safe.   If we need to, for some reason, we can either hotfix the existing tag or restore a local `LruCache` copy here.

If we haven't, can we please open an issue to swap this over?   I don't want to forget since it's not a package with work happening otherwise.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/37778

Reply:
Would this also be true about having the Azure.Data.SchemaRegistry project reference when it is still a beta?

Reply:
Yes.   We wouldn't be able to release the Avro package to GA if it depended on beta features in the SR package.